job-service-dropwizard/src/main/resources/config.yaml

```diff
@@ -25,3 +25,47 @@ server:
 
 logging:
   type: external
+
+health:
+  healthChecks:
+    - name: deadlocks
+      type: alive
+      critical: true
+      initialState: false
+      schedule:
+        initialDelay: ${JOB_SERVICE_LIVENESS_INITIAL_DELAY_DURATION:-15s}
+        checkInterval: ${JOB_SERVICE_LIVENESS_CHECK_INTERVAL_DURATION:-60s}
+        downtimeInterval: ${JOB_SERVICE_LIVENESS_DOWNTIME_INTERVAL_DURATION:-60s}
+        failureAttempts: ${JOB_SERVICE_LIVENESS_FAILURE_ATTEMPTS:-3}
+        successAttempts: ${JOB_SERVICE_LIVENESS_SUCCESS_ATTEMPTS:-1}
+    - name: database
+      type: ready
+      critical: true
+      initialState: false
+      schedule:
+        initialDelay: ${JOB_SERVICE_READINESS_INITIAL_DELAY_DURATION:-15s}
+        checkInterval: ${JOB_SERVICE_READINESS_CHECK_INTERVAL_DURATION:-60s}
+        downtimeInterval: ${JOB_SERVICE_READINESS_DOWNTIME_INTERVAL_DURATION:-60s}
+        failureAttempts: ${JOB_SERVICE_READINESS_FAILURE_ATTEMPTS:-3}
+        successAttempts: ${JOB_SERVICE_READINESS_SUCCESS_ATTEMPTS:-1}
+    - name: ping
+      type: ready
+      critical: true
+      initialState: false
+      schedule:
+        initialDelay: ${JOB_SERVICE_READINESS_INITIAL_DELAY_DURATION:-15s}
+        checkInterval: ${JOB_SERVICE_READINESS_CHECK_INTERVAL_DURATION:-60s}
+        downtimeInterval: ${JOB_SERVICE_READINESS_DOWNTIME_INTERVAL_DURATION:-60s}
+        failureAttempts: ${JOB_SERVICE_READINESS_FAILURE_ATTEMPTS:-3}
+        successAttempts: ${JOB_SERVICE_READINESS_SUCCESS_ATTEMPTS:-1}
+    - name: queue
+      type: ready
+      critical: true
+      initialState: false
+      schedule:
+        initialDelay: ${JOB_SERVICE_READINESS_INITIAL_DELAY_DURATION:-15s}
+        checkInterval: ${JOB_SERVICE_READINESS_CHECK_INTERVAL_DURATION:-60s}
+        downtimeInterval: ${JOB_SERVICE_READINESS_DOWNTIME_INTERVAL_DURATION:-60s}
+        failureAttempts: ${JOB_SERVICE_READINESS_FAILURE_ATTEMPTS:-3}
+        successAttempts: ${JOB_SERVICE_READINESS_SUCCESS_ATTEMPTS:-1}
+        
```

Comment:
Too many newlines?

Reply:
The newline is being strange, this is just with one newline but it doesn't seem to be picking it up correctly. Can take this off? Then there will just be no newline. Or if you have any suggestions on how to make the newline get picked up?

Reply:
Think it was a tab. fixed now

Reply:
Ah thanks very much